lib/wolfssl/atca_wolfssl_interface.c

```diff
@@ -413,11 +413,11 @@ ATCA_STATUS atcac_pk_init_pem(
             type = ECC_PUBLICKEY_TYPE;
         }
 
-        ret = PemToDer((unsigned char*)buf, (long)buflen, type, &der, NULL, NULL, &ecckey);
+        ret = PemToDer((const unsigned char*)buf, (long)buflen, type, &der, NULL, NULL, &ecckey);
```

Comment:
Casting to const before passing to a function doesn't make sense for C code - What warning is being triggered here?